brave-cxf/src/main/java/com/github/kristofa/brave/cxf/MessageWrapper.java

```diff
@@ -0,0 +1,64 @@
+package com.github.kristofa.brave.cxf;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.apache.cxf.message.Message;
+
+/**
+ * @author Michał Podsiedzik
+ */
+public class MessageWrapper {
```

Comment:
package private. one-line description of why this is here. It seems to be http in nature, so should it be HttpMessageWrapper?